apps/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "../tsconfig.base.json",
+  "include": [
+    "./legacy-api/src",
+    "./ocean-api/src",
+    "./rich-list-api/src",
+    "./libs/*/src"
+  ],
+  "compilerOptions": {
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "outDir": "./dist",
+    "baseUrl": "./",
+    "paths": {
+      "@defichain-app-lib/actuator": [
+        "libs/actuator/src"
+      ],
+      "@defichain-app-lib/actuator/*": [
+        "libs/actuator/src/*"
+      ],
```

Comment:
```suggestion
      "@defichain-app-lib/*": [
        "libs/*/src"
      ],
```
Wonder if we want to keep the simplicity of maintaining this file like we do with shared packages. Since this is within the scope of shared packages this space will dynamically grow as we abstract out shared functionality.

Reply:
Yea I fully agree

But removing this causes `build` to fail - typescript needs to know how to resolve the imports
![image](https://user-images.githubusercontent.com/31790206/154893011-95adb488-d805-4317-84f0-5803e010778c.png)

I was also trying to figure out how best to avoid duplicating this across `jellyfish/tsconfig.json` and `jellyfish/apps/tsconfig.json`, but couldn't find an elegant way so far 🤔

Reply:
Woopsies - solution is for jellyfish/apps/tsconfig.json to extend jellyfish/tsconfig.json

Reply:
Generalised to `'@defichain-app-lib/(.*)': '<rootDir>/apps/libs/$1/src'`
Thanks!